includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -188,202 +196,350 @@ class DesignSystemGlobalFooterModel extends WikiaModel {
 		]
 	];
 
-	private $wikiId;
-	private $lang;
-
-	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
-		parent::__construct();
-
-		$this->wikiId = $wikiId;
-		$this->lang = $lang;
-	}
-
-	public function setHrefs( $hrefs ) {
-		$this->hrefs = $hrefs;
-	}
-
-	public function getData() {
-		$data = [
-			'company_overview' => [
-				'header' => [
-					'type' => 'line-text',
+	private $baseData = [
```

Comment:
Schema describes the shape of data so `$schemaData` doesn't really fit here. `$baseData` sounds fine to me.